renderdoccmd/renderdoccmd_android.cpp

```diff
@@ -77,11 +78,15 @@ void handle_cmd(android_app *app, int32_t cmd)
       __android_log_print(ANDROID_LOG_INFO, LOGCAT_TAG,
                           "APP_CMD_INIT_WINDOW: android_state->window: %p", android_state->window);
 
-      const char *argv[] = {
-          "renderdoccmd", "replay", "/sdcard/capture.rdc",
+      char prop[PROP_VALUE_MAX];
+      if(GetAndroidProperty("renderdoc.replay", prop) > 0)
```

Comment:
The purpose was just to specify the input filename to replay. After looking at this again there is a better way of doing this that doesnt need setprop.